src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -246,6 +256,26 @@ class Internals(SimpleNamespace):
             short_description="Specify if the editor is console based",
             value=EntryValue(default=True),
         ),
+        Entry(
+            name="exec_command",
+            cli_parameters=CliParameters(short="--excmd"),
+            settings_file_path_override="exec.command",
+            short_description="Specify the command to run within the execution environment",
+            subcommands=["exec"],
+            value=EntryValue(default="/bin/bash"),
+        ),
+        Entry(
+            name="exec_shell",
+            choices=[True, False],
+            cli_parameters=CliParameters(short="--exshell"),
```

Comment:
The configuration subsystem doesn't really support simple flags without a value, the reason is because of the way we generate an environment variable and settings file entry.   There would be some inconsistency between the CLI, env var and settings file.  Good or bad, the way it works today is all three are a bool and require true/false.

I prefixed it with `exec_` to match `exec_command`, let me know if you feel strongly you feel about this one, I can change if you do.

Reply:
@relrod thoughts here?

Reply:
@relrod ping!

Reply:
@relrod what do you think?

Reply:
I don't feel super strongly about it, I mainly am just going by what other CLI apps tend to do, but like I mentioned, we're already deviating from the norm by requiring an explicit `true` or `false` anyway.